docker-compose.yml

```diff
@@ -173,10 +173,16 @@ services:
     ports:
       - 3000:3000
     volumes:
-      - ./website/blog:/workdir/docusaurus-docs/blog:ro
       - ./website/docs:/workdir/docusaurus-docs/docs:ro
-      - ./website/src:/workdir/docusaurus-docs/src:ro
       - ./website/static:/workdir/docusaurus-docs/static:ro
+      - ./website/src/components:/workdir/docusaurus-docs/src/components:ro
+      - ./website/src/css:/workdir/docusaurus-docs/src/css:ro
+
+      # We don't copy src/pages/index.js for docs as we have an index page defined through slug
+      # However, we copy intro.js to redirect from /intro to the root docs page.
```

Comment:
We agreed with @Fashander that a special landing page is not needed for docs, so `intro` should become our landing page.